wordpress/wp-content/themes/cds-default/style.css

```diff
@@ -6,3 +6,209 @@
  * Domain Path: /languages
  * Version: 1.8.0
 */
+
+/* https://jsonformatter.org/sass-to-css */
+
+/*
+$colors: (
```

Comment:
Noting: I didn't want to add a SASS build step in this PR so the commented out code is what is used to generate the  CSS at the bottom of the file.  Using this tool for now https://jsonformatter.org/sass-to-css